lib/publiccloud/provider.pm

```diff
@@ -39,9 +39,9 @@ has ssh_key => get_ssh_private_key_path();
 =cut
 
 sub init {
-    my ($self) = @_;
+    my ($self, %args) = @_;
```

Comment:
we actually need just ONE argument . I think it would be more readable if we will keep it this way
```suggestion
    my ($self, skip_ssh_local_config) = @_;
```

Reply:
This would mean the caller in `lib/publiccloud/basetest.pm` would need to be changed from `$provider->init(%args)` to something like `$provider->init($args{skip_ssh_local_config})`, which will have references to this `skip_ssh_local_config` variable in 3 places (basetest, provider and in the calling module). Is that OK? If it is, I'll push changes.

Reply:
As you may see, I need to also change the calls to `$self->SUPER::init()` in many files to pass `%args` to `publiccloud::provider::init()`, so I'm leaving this with `%args` as it seems easier to extend and doesn't add anything about skipping the generation of `.ssh/config` in all child classes, leaving a reference to this only in `publiccloud::provider` itself.